Applications/Spire/Include/Spire/Ui/TableModel.hpp

```diff
@@ -135,56 +116,6 @@ namespace Details {
   const T& TableModel::get(int row, int column) const {
     return std::any_cast<const T&>(at(row, column));
   }
```

Comment:
Add back this function:

    template<typename... F>
    void visit(const TableModel::Operation& operation, F&&... f) {
      return Details::visit<TableModel>(operation, std::forward<F>(f)...);
    }